### DISCUSSION TITLE
use CK API

### DISCUSSION THREAD

**Discussion by xiaobo1025** [ROCm/ROCm#2821] [Q&A] [answered]
Dear friends, I'm going to use CK's API to write a gemm.cpp program for matrix multiplication. How should I implement this task?Thank you very much!

**Comment by huanrwan-amd** [comment, marked as answer]
Hi @xiaobo1025, thank you for your question.
For composable_kernel, you can refer to [composable_kernel](https://github.com/ROCm/composable_kernel). 

- Hello World Tutorial: https://rocm.docs.amd.com/projects/composable_kernel/en/latest/tutorial/tutorial_hello_world.html
- Examples for your reference: https://github.com/ROCm/composable_kernel/tree/develop/example
If you have any further questions, please let us know.
